portfolio/src/main/webapp/scripts/login.js

```diff
@@ -3,9 +3,16 @@
  */
 async function showCommentFeedback() {
```

Comment:
better name please.  The meaning of "Feedback" is not clear, and this method seems to deal entirely with logging in and not with comments.